### PR TITLE
Cards black list on game creation form

### PR DESCRIPTION
**Pull request by alrusdi**
Ability to exclude some cards from the game
Please check my english too
cards_black_list

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -371,6 +371,7 @@ function createGame(req: http.IncomingMessage, res: http.ServerResponse): void {
         showOtherPlayersVP: gameReq.showOtherPlayersVP,
         customCorporationsList: gameReq.customCorporationsList,
         customColoniesList: gameReq.customColoniesList,
+        cardsBlackList: gameReq.customColoniesList,
```
[the remark this answers is not in the document]
Comment (alrusdi) [reply server.ts]: Sad typo. Fixed

**Comment by vincentneko** [comment]
Thanks for the update.
When only one card has been added to the list, the orange button with the cross to remove it is not working.

**Comment by alrusdi** [comment]
> Thanks for the update.
> When only one card has been added to the list, the orange button with the cross to remove it is not working.

Fixed

**Review by vincentneko — APPROVED** [review]
Great addition !

**Comment by samyelias** [comment]
It would be convenient to be able to import a text file of excluded cards if you have a favourite setup, so you don't have to type them all in again every time.

**Pull request merged**